src/components/modal/v2/styles/components/_content.scss

```
@@ -133,9 +137,25 @@
                 margin-top: 28px;
             }
```

```suggestion
            }
            
            &.checkout {
              @include mobile {
                   margin-bottom: 60px;
               }
          }
```

Small tweak here to the disclosure.